### PR TITLE
Add Android roadmap

### PR DESCRIPTION
**Pull request by jin**
cc @laurentlb

### CODE REVIEW

**Comment by laurentlb** [comment]
Cool, thanks!

File: roadmaps/android.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# Android Roadmap
+
+*Last verified: 2018-07-21*
+([update history](https://github.com/bazelbuild/bazel-website/commits/master/roadmaps/android.md))
+
+**Points of contact:** [ahumesky](https://github.com/ahumesky), [jingwen](https://github.com/jin)
+
+## Overview
+
+Bazel is capable of building Android apps using the [Android
```
Comment (ahumesky) [inline roadmaps/android.md]: nit: "can build"

File: roadmaps/android.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+rules](https://docs.bazel.build/versions/master/be/android.html). Blaze, the
+Google build tool that Bazel is open-sourced as, is used to build Google apps
```
Comment (ahumesky) [inline roadmaps/android.md]: nit: "Blaze, Google's internal version of Bazel"

File: roadmaps/android.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+and [instrumentation
+testing](https://docs.bazel.build/versions/master/android-instrumentation-test.html).
+
+## Goal
+
+Build and test Android apps of all sizes quickly and reliably.
+
+## Roadmap 
+
+As of Q3 2018, the native Android rules are incrementally rewritten and migrated
```
Comment (ahumesky) [inline roadmaps/android.md]: nit: "are being incrementally rewritten ...."

File: roadmaps/android.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+#### Developer experience
+
+- Document Android with Bazel compatibility across Windows, macOS and Linux.
+
+#### NDK
+
+- Document Android NDK support with Bazel.
+
+#### Tooling (Build & Test)
```
Comment (ahumesky) [inline roadmaps/android.md]: you can add "Support Databinding 3.2.0 (v2)"

File: roadmaps/android.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+development of the Android rules can be decoupled from the core build system
+development.
+
+### Q3 2018
+
+#### Build performance
+
+- Publish a prototype continuous build performance dashboard for Android projects
+
+#### Developer experience
```
Comment (ahumesky) [inline roadmaps/android.md]: you can add something about tools_android: "Add `tools_android` repository for common android build tools for use with bazel (e.g. Firebase, Google Services)

Comment (jin) [reply roadmaps/android.md]: done

File: roadmaps/android.md (hunk 4 above)
Comment (jin) [reply roadmaps/android.md]: done

File: roadmaps/android.md (hunk 3 above)
Comment (jin) [reply roadmaps/android.md]: done

File: roadmaps/android.md (hunk 2 above)
Comment (jin) [reply roadmaps/android.md]: done

File: roadmaps/android.md (hunk 1 above)
Comment (jin) [reply roadmaps/android.md]: done

File: roadmaps/android.md [hunk 6]
```diff
@@ -38,20 +38,24 @@ This roadmap is a high level overview of planned features and fixes. See the
...
 
-- Publish a prototype continuous build performance dashboard for Android projects
+- Publish a prototype continuous build performance dashboard for Android projects.
 
 #### Developer experience
 
 - Document Android with Bazel compatibility across Windows, macOS and Linux.
+- Introduce a best-practices documentation on configuring your Android build.
+- Introduce `tools_android` repository for common Android build tools for use
+  with Bazel (e.g. Firebase, Google Play Services)
```
Comment (ahumesky) [inline roadmaps/android.md]: in this case it's just "Google Services". "Google Play Services" is a different thing.

**Pull request merged**